### PR TITLE
format amount on blur

### PR DESCRIPTION
**Pull request by Kamill90**

### CODE REVIEW

File: src/screens/FilterTransactionsScreen.tsx [hunk 1]
```diff
@@ -183,6 +182,12 @@ class FilterTransactionsScreen extends PureComponent<Props, State> {
                 label={i18n.filterTransactions.fromAmount}
                 suffix="BTCV"
                 keyboardType="numeric"
+                onBlur={() => {
+                  !this.validateAmount(fromAmount) &&
```
Comment (PanRadaCloudbest) [inline src/screens/FilterTransactionsScreen.tsx:186]: can we move this to the top function?

File: src/screens/FilterTransactionsScreen.tsx [hunk 2]
```diff
@@ -192,6 +197,10 @@ class FilterTransactionsScreen extends PureComponent<Props, State> {
                 label={i18n.filterTransactions.toAmount}
                 suffix="BTCV"
                 keyboardType="numeric"
+                onBlur={() => {
+                  !this.validateAmount(toAmount) &&
+                    this.props.updateToAmount(i18n.formatBalanceWithoutSuffix(Number(toAmount), '', true).toString());
```
Comment (PanRadaCloudbest) [inline src/screens/FilterTransactionsScreen.tsx:202]: same

File: src/screens/FilterTransactionsScreen.tsx (hunk 1 above)
Comment (Kamill90) [reply src/screens/FilterTransactionsScreen.tsx]: done

File: src/screens/FilterTransactionsScreen.tsx (hunk 2 above)
Comment (Kamill90) [reply src/screens/FilterTransactionsScreen.tsx]: done

File: src/screens/FilterTransactionsScreen.tsx [hunk 3]
```diff
@@ -132,9 +132,22 @@ class FilterTransactionsScreen extends PureComponent<Props, State> {
       : this.transactionTagsSent;
   }
 
+  formatFromAmount = () => {
```
Comment (KedziaPawel) [inline src/screens/FilterTransactionsScreen.tsx:135]: these functions are nearly identical, I would create a function `formatAmount(amount, update)`

Comment (Kamill90) [reply src/screens/FilterTransactionsScreen.tsx:135]: done

File: src/screens/FilterTransactionsScreen.tsx [hunk 4]
```diff
@@ -132,9 +132,27 @@ class FilterTransactionsScreen extends PureComponent<Props, State> {
...
```
Comment (KedziaPawel) [inline src/screens/FilterTransactionsScreen.tsx:135]: to delete

File: src/screens/FilterTransactionsScreen.tsx [hunk 5]
```diff
@@ -132,9 +132,27 @@ class FilterTransactionsScreen extends PureComponent<Props, State> {
...
+    const { fromAmount } = this.props.filters;
+
+    !this.validateAmount(fromAmount) &&
+      this.props.updateFromAmount(i18n.formatBalanceWithoutSuffix(Number(fromAmount), '', true).toString());
+  };
+
+  formatToAmount = () => {
```
Comment (KedziaPawel) [inline src/screens/FilterTransactionsScreen.tsx:142]: to delete

Comment (Kamill90) [reply src/screens/FilterTransactionsScreen.tsx]: done

**Review by KedziaPawel — APPROVED** [review]

**Pull request merged**